var/spack/repos/builtin/packages/aocl-sparse/package.py

```diff
@@ -29,8 +29,10 @@ class AoclSparse(CMakePackage):
             values=('Debug', 'Release'))
     variant('shared', default=True,
             description='Build shared library')
+    variant('ilp64', default=False,
+            description='Build with ILP64 support')
 
-    depends_on('boost')
+    depends_on('boost', when='@2.2')
```

Comment:
Again, does this dependency only apply to version `2.2`.

Reply:
Yes, Boost is required only for aocl-sparse 2.2 version.